plugins/jvm-abi-gen/src/org/jetbrains/kotlin/jvm/abi/JvmAbiOutputExtension.kt

```diff
@@ -76,14 +78,17 @@ class JvmAbiOutputExtension(
 
                     else -> /* abiInfo is AbiClassInfo.Stripped */ {
                         val methodInfo = (abiInfo as AbiClassInfo.Stripped).methodInfo
-                        val innerClassInfos = mutableMapOf<String, InnerClassInfo>()
                         val innerClassesToKeep = mutableSetOf<String>()
                         val writer = ClassWriter(0)
                         val remapper = ClassRemapper(writer, object : Remapper() {
                             override fun map(internalName: String): String =
                                 internalName.also { innerClassesToKeep.add(it) }
                         })
                         ClassReader(outputFile.asByteArray()).accept(object : ClassVisitor(Opcodes.API_VERSION, remapper) {
+                            private val keptFields = mutableListOf<FieldNode>()
+                            private val keptMethods = mutableListOf<MethodNode>()
+                            private val innerClassInfos = mutableMapOf<String, InnerClassInfo>()
```

Comment:
```suggestion
                            private val innerClassInfos = mutableMapOf<ClassName, InnerClassInfo>()
```

Wouldn't it not make sense to not keep this as a raw `String` but rather declare a well named `typealias` wouldn't it?

Reply:
On top of that, I think the name should indicate that it's ONLY ABI inner classes all of which will be included into the ABI later on line 161.

`abiInnerClassInfos`

Reply:
Considering that the scope of the property is pretty narrow I think it is not worth it. Nonetheless even using `ClassName` typealias or value class wont add a lot of readability here since it doesn't tell the reader if it is FQCN or short name or canonical name :)

Reply:
IMHO `String` is fine here for now